benchmark/jsonschema/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_executable(sourcemeta_jsontoolkit_benchmark_jsonschema compiler_basic.cc)
+
+target_link_libraries(sourcemeta_jsontoolkit_benchmark_jsonschema benchmark::benchmark)
```

Comment:
You are missing the visibility attribute here. I guess all of them should be `PRIVATE`